### DISCUSSION TITLE
Degraded performance after compositor reloading?

### DISCUSSION THREAD

**Discussion by mock1328** [hyprwm/Hyprland#7670] [Q&A]
After several compositor reloads (either with `hyprctl reload` or just by updating config files) opening new apps/closing existing causes notable freezes for what it feels like ~1 second. Is this a known issue?

**Comment by Aqa-Ib** [comment]
Never happened to me before. Might be a hard disk issue?

**Reply by mock1328** [reply]
I'm not sure, but this wasn't a problem somewhere around 0.39 version if I'm not mistaken, only in later ones

**Reply by Aqa-Ib** [reply]
You can try downgrading to see if it is still working in that version
